Rebecas_MD_ex.md

```diff
@@ -1,13 +1,13 @@
 # **Rebeca's Markdown exercise**
```

Comment:
```suggestion
# Rebeca's Markdown Exercise
```

Remember title case. Also, at least in Markdown, we normally don't add boldface to titles. The hash symbol (#) by itself changes the formatting of the text so that it is noticeable that a heading is different from the body text.